apps/avifenc.c

```diff
@@ -76,12 +76,27 @@ static void syntaxShort(void)
 static void syntaxLong(void)
 {
     printf("Syntax: avifenc [options] input.[jpg|jpeg|png|y4m] output.avif\n");
-    printf("Options:\n");
+    printf("Standard options:\n");
     printf("    -h,--help                         : Show short syntax help\n");
     printf("    --longhelp                        : Show long syntax help\n");
```

Comment:
Optional: `(this page)` could be mentioned here for extra clarity

Reply:
Done